### DISCUSSION TITLE
Getting error

### DISCUSSION THREAD

**Discussion by GraemRus** [smy20011/dreambooth-gui#68] [Q&A] [answered]
v0.1.9: Pulling from smy20011/dreambooth
Digest: sha256:50f11f2b6d7a0d5fba718eea0858367cb82840ee6847e5422e42aed1c7921653
Status: Image is up to date for smy20011/dreambooth:v0.1.9
docker: Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error running hook #0: error running hook: exit status 1, stdout: , stderr: Auto-detected mode as 'legacy'
nvidia-container-cli: initialization error: WSL environment detected but no adapters were found: unknown.

__________

What wrong and how can I fix it?

**Comment by imakefun** [comment, marked as answer]
I'm also having this issue. I'm completely new to all things ML, AI, and SD. 
I've never installed docker previously, had to also install the CUDA toolkit, and various other things.. but have googled for several hours trying to diagnose this issue to no avail. 

It might be an obvious question, but do we need to install a linux distro on Docker?

**Reply by imakefun** [reply]
Pretty sure I found the answer from the reddit thread. You need to run windows update and get the absolute latest version (22h2)

![image](https://user-images.githubusercontent.com/12904870/202875827-d7baf45d-2442-4cdd-aa28-b9fa9b1114ef.png)
